.github/workflows/ci.yml

```diff
@@ -49,8 +48,6 @@ jobs:
             ${{ runner.os }}-build-
             ${{ runner.os }}-
 
-      - name: Install npm ${{ env.NPM_VERSION }}
```

Comment:
Was the motivation here just that it wasn't working (presumably because 7.x wasn't available on 14/16)? 

I'm fine removing it, doesn't seem like it has much impact other than to use the default version of npm installed with node and I can't think of why that would really matter. Just wanted to make sure I understood ramifications.

Reply:
I think we're fine here.